README.md

```diff
@@ -85,21 +85,12 @@ You can also create an Azure service principal either through
     git clone https://github.com/Azure-Samples/azure-sdk-for-python-keyvault-secrets-get-set-managedid.git
     ```
 
-4.  Run the following command to install dependencies for "SDK version 3" and "SDK version 4":
+4.  Run the following command to install dependencies:
 
-- SDK version 4
 
-```
-cd v4
-pip install -r requirements.txt
-```
-
-- SDK version 3
-
-```
-cd v3
-pip install -r requirements.txt
-```
+    ```
+    pip install -r requirements.txt
+    ```
 
 5.  Set up the environment variable `KEY_VAULT_URL` with your KeyVault URL or replace the variable in the example file.
```

Comment:
`examply.py` attempts to get `KEY_VAULT_URI` from the environment, so this will need to align with that:
```suggestion
5.  Set up the environment variable `KEY_VAULT_URI` with your KeyVault URL or replace the variable in the example file.
```